README.rst

```diff
@@ -6,8 +6,8 @@ git_wrapper
 .. image:: https://img.shields.io/pypi/v/git_wrapper.svg
         :target: https://pypi.python.org/pypi/git_wrapper
 
-.. image:: https://img.shields.io/travis/fuzzball81/git_wrapper.svg
-        :target: https://travis-ci.org/fuzzball81/git_wrapper
+.. image:: https://img.shields.io/travis/release-depot/git_wrapper.svg
+        :target: https://travis-ci.org/release-depot/git_wrapper
```

Comment:
This change looks to already be in master?  I recall you fixing this in the last patchset